flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/runtime/operators/aggregate/window/combines/CombineRecordsFunction.java

```diff
@@ -54,8 +57,10 @@
     /** Function used to handle all aggregates. */
     private final NamespaceAggsHandleFunction<Long> aggregator;
 
-    /** Whether to copy input key, because key is reused. */
-    private final boolean requiresCopyKey;
+    /** Whether to copy input record, because record is reused. */
```

Comment:
```suggestion
    /** Whether to copy key and input record, because key and record are reused. */
```

We should also mention key, because key and record are separate.